### PR TITLE
Update

### PR DESCRIPTION
**Pull request by TuvalSimha**

### CODE REVIEW

File: src/fets/client.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { type NormalizeOAS, createClient } from "fets";
+import openapi from "./openapi";
```
Comment (ardatan) [inline src/fets/client.ts:2]:
```suggestion
import type openapi from "./openapi";
```

File: src/helpers.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
+export interface Player {
```
Comment (ardatan) [inline src/helpers.ts:1]: You can use `OASModel` from feTS.

Comment (TuvalSimha) [reply src/helpers.ts:1]: like that also make a errors:

```
type Player = OASModel<
  NormalizeOAS<(typeof openapi)["definitions"]["Player"]["properties"]>,
  "Player"
>;
```

@ardatan

Comment (ardatan) [reply src/helpers.ts:1]: Could you try with `fets@0.4.8`?

Comment (TuvalSimha) [reply src/helpers.ts]: @ardatan 
Its work!!! thank you Arda❤️

**Pull request merged**